test/integration/vaultauthmethods/terraform/main.tf

```diff
@@ -15,6 +15,10 @@ terraform {
       source  = "hashicorp/helm"
       version = "2.11.0"
     }
+    google = {
```

Comment:
I don't see where this is being used. Is it still needed?

Reply:
Nope, not needed anymore.